docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`5824` Addresses registered and attributes set via new ENS smart contracts will be correctly shown.
```

Comment:
We already do that right? Shouldn't this be more about extending support for ENS? something like

```suggestion
* :feature:`5824` Improved support for ENS allowing to decode latest version of their contracts.
```